vignettes/articles/multi-scale-resampling.Rmd

```diff
@@ -0,0 +1,233 @@
+---
+title: "Multi-scale model assessment with spatialsample"
+---
+
+```{r include = FALSE}
+knitr::opts_chunk$set(
+  cache = TRUE,
+  cache.lazy = FALSE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+Modeling spatially structured data is complicated. In addition to the usual difficulty of statistical modeling, models of spatially structured data may have spatial structure in their errors, with different regions being more or less well-described by a given model. This also means that accuracy metrics for these models might change depending on what spatial scale is being assessed. Only investigating model accuracy at larger aggregation scales, such as when accuracy is only assessed for the entire study area as a whole, might "smooth out" these local differences and present an inaccurate picture of model performance.
+
+For this reason, a number of researchers (most notably, [Riemann et al. (2010)](https://www.nrs.fs.fed.us/pubs/jrnl/2010/nrs_2010_riemann_001.pdf)[^1]) have suggested assessing models at multiple scales of spatial aggregation to ensure cross-scale differences in model accuracy are identified and reported. This post walks through how to do that using the new [spatialsample](https://spatialsample.tidymodels.org/) package.
+
+[^1]: Riemann, R., Wilston, B. T., Lister, A., and Parks, S. 2010. An effective assessment protocol for continuous geospatial datasets of forest characteristics using USFS Forest Inventory and Analysis (FIA) data. Remote Sensing of Environment, 114, pp. 2337-2353. doi: 10.1016/j.rse.2010.05.010.
```

Comment:
Unfortunately 😭 footnotes don't work on tidymodels.org so if you can work to integrate any references into more like a references section at the end, and any other content into the text itself, that would be great.